From 5ba26384aaa06dea8f2573e045dda9e426641542 Mon Sep 17 00:00:00 2001
From: Zhi Wang <zhi.a.wang@intel.com>
Date: Mon, 27 Apr 2015 05:18:56 +0800
Subject: [PATCH 053/403] Revert "vgt: remove 0x20e4 from register switch list
 as we have set the correct value in i915."

This reverts commit 90e4485ad7b7b1766052e5263ebd8eac667a7947.
---
 drivers/gpu/drm/i915/vgt/render.c |    1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/gpu/drm/i915/vgt/render.c b/drivers/gpu/drm/i915/vgt/render.c
index ab4382e..a5d8749 100644
--- a/drivers/gpu/drm/i915/vgt/render.c
+++ b/drivers/gpu/drm/i915/vgt/render.c
@@ -1319,6 +1319,7 @@ static struct reg_mask_t rcs_reset_mmio[] = {
 	{0x2044, 0},
 
 	{0x20a0, 0},
+	{0x20e4, 1},
 	{0x7004, 1},
 	{0x20dc, 1},
 
-- 
1.7.10.4

